control/control_performance_analysis/README.md

```diff
@@ -0,0 +1,28 @@
+# control_performance_analysis
+
+## Purpose
+
+`control_performance_analysis` is the package to analyze the output of a control module.
```

Comment:
`control_performance_analysis` is the package to analyze the tracking performance of a control module.

Reply:
```suggestion
`control_performance_analysis` is the package to analyze the tracking performance of a control module.
```